tests/utlis/server/playground.ts

```diff
@@ -0,0 +1,52 @@
+import { expect, Page } from '@playwright/test';
+
+type ScriptOptionsKeys = 'version' | 'server' | 'selector';
+type ScriptOptions = {
+  [key in ScriptOptionsKeys]?: string;
+};
+function getOptions(options: ScriptOptions = {}) {
+  return Object.entries(options)
+    .filter(([, val]) => val)
+    .map(([key, val]) => `data-${key}="${val}"`)
+    .join(' ');
+}
+
+export async function gotoHtmlWidget(page: Page, html: string): Promise<Page>;
+export async function gotoHtmlWidget(
+  page: Page,
+  config: ScriptOptions,
+  html: string,
+): Promise<Page>;
```

Comment:
Are they duplicates?

Reply:
🤔 no. There is overload - with two or three parameters. Or what do you mean?

Reply:
I'm trying to understand why not just mark the configuration as optional param and have one function. 
Does it seem like overloading is type-safer? 
Then, it is good move, no questions.